website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step12-add-nodepool.sh

```diff
@@ -49,6 +49,7 @@ spec:
   amiSelectorTerms:
     - id: "${ARM_AMI_ID}"
     - id: "${AMD_AMI_ID}"
-#   - id: "${GPU_AMI_ID}" # <- GPU Optimized AMD AMI 
-#   - name: "amazon-eks-node-${K8S_VERSION}-*" # <- automatically upgrade when a new AL2 EKS Optimized AMI is released. This is unsafe for production workloads. Validate AMIs in lower environments before deploying them to production.
+#   - id: "${NVIDIA_AMI_ID}" # <- AL2023 NVIDIA AMI 
+#   - id: "${NEURON_AMI_ID}" # <- AL2023 NEURON AMI
+#   - name: "amazon-eks-node-${K8S_VERSION}-*" # <- automatically upgrade when a new AL2023 EKS Optimized AMI is released. This is unsafe for production workloads. Validate AMIs in lower environments before deploying them to production.
```

Comment:
Should we make warning in the AMIFamily section that AL2023 will not be supported in 1.33? Seems like this would be something we should in just the preview section